### PR TITLE
Test gcp with secure boot 4.13 periodics

### PR DESCRIPTION
**Pull request by eranco74**

### CODE REVIEW

**Comment by eranco74** [comment]
/pj-rehearse
/pj-rehearse periodic-ci-openshift-release-master-ci-4.13-e2e-gcp-secureboot

**Comment by eranco74** [comment]
/pj-rehearse periodic-ci-openshift-release-master-ci-4.13-e2e-gcp-secureboot

**Comment by omertuc** [comment]
/pj-rehearse periodic-ci-openshift-release-master-ci-4.13-e2e-gcp-secureboot

File: ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-commands.sh [hunk 1]
```diff
@@ -98,3 +98,20 @@ EOF
   yq-go m -x -i "${CONFIG}" "${PATCH}"
   rm "${PATCH}"
 fi
+
+if ! [[ -z "${SECURE_BOOT}" ]]; then
```
Comment (vrutkovs) [inline ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-commands.sh:102]: Lets move this to a separate step to avoid affecting a lot of tests

**Comment by eranco74** [comment]
closing in favor of https://github.com/openshift/release/pull/35686

**Pull request closed without merging**